scrapers/RawFuckClub.yml

```diff
@@ -0,0 +1,28 @@
+name: RawFuckClub

```

Comment:
You've used `RawFuckClub` here but `Raw Fuck Club` as the fixed studio name: this isn't visible in the Stash UI since there's no query scraper here, but it will probably be visible in the new scraper management UI that's (probably) coming in v0.24

Just a nitpick 👍

Reply:
Hey @Maista6969, Thanks for the feedback! Why don't you go ahead and push it as I'm a little tapped out right now with my recent script "hack" job LOL. I can take a look at these changes later.

Noted on the script/studio naming convention.

With the kinkmen url, my thinking there was: what happens when they start to update the newer URL in that field? Need some if/then logic if post processing and quite frankly at this point I don't know exactly how to do that LOL so I need to take a moment to research. I also haven't had a chance to take a look to see if the newer scenes contain then you were kinkmen.com URL.

Also this being my first ever fork/pull request, after you push this to master, I don't have to do anything? And do I continue to use this fork I've created for additional updates that I may contribute? I notice other forks showed not directly mergeable so I didn't know if there's something I have to do afterwards.

Thanks so much!